scripts/spark_eval/run_gcs_cluster.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# Starts a GCS cluster, runs scripts, then deletes the cluster.
+
+if [ -z "$GCS_CLUSTER" ]; then
+  echo "Please set the GCS_CLUSTER environment variable to the name of the cluster you would like to start."
+  exit 1
+fi
+
+if [ -z "$API_KEY" ]; then
+  echo "Please set the API_KEY environment variable to a path to the local json file."
+  exit 1
+fi
+
+# Create cluster
+gcloud dataproc clusters create "$GCS_CLUSTER" \
```

Comment:
@tomwhite Lets make these auto-deleting clusters now that that's an option.  You can do it by switching to

`gcloud beta dataproc clusters create ...` and adding the new `--max-idle` and `--max-age` parameters.

Reply:
Done